Makefile

```diff
@@ -489,10 +494,31 @@ verify-links:
 	fi;
 .PHONY: verify-links
 
-verify: verify-gofmt verify-codegen verify-crds verify-helm-schemas verify-helm-charts verify-deploy verify-examples verify-govet verify-helm-lint verify-links
+# $1 - source file
+# $2 - client directory
+# $3 - client name
+define generate-swagger-client
+	GOPATH=$(GOPATH) $(GO) run "$(SWAGGER)" generate client --name "$(3)" --spec "$(1)" --target "$(2)"
+
+endef
+
+update-manager-client:
+	$(call generate-swagger-client,${MANAGER_CLIENT_SPEC},${MANAGER_CLIENT_GEN_DIR},'manager')
+.PHONY: update-manager-client
+
+verify-manager-client: tmp_dir := $(shell mktemp -d)
+verify-manager-client: MANAGER_CLIENT_TMP_DIR := "$(tmp_dir)/${MANAGER_CLIENT_GEN_DIR}"
```

Comment:
this ends up double quoted as well

Reply:
nit: i'd drop the extra variable as using it explicitelly makeses sure e..g you are diffing the same folder with just different prefixes